Makefile

```diff
@@ -103,3 +103,12 @@ binaries-all-sys:
 	GOOS=linux   GOARCH=amd64 $(MAKE) binaries
 	GOOS=linux   GOARCH=arm64 $(MAKE) binaries
 	GOOS=windows GOARCH=amd64 $(MAKE) binaries
+
+.PHONY: update-otel
+update-otel:
+	for mod in $$(grep -l 'github.com/open-telemetry/opentelemetry-collector ' $$(find . -name go.mod)); do \
+		dir=$$(dirname $$mod); \
+		(cd $$dir && go get github.com/open-telemetry/opentelemetry-collector@master); \
```

Comment:
suggestion: use a conditional variable assignment to make easy to set up other targets

Reply:
Do you mean for other dependencies we might want to mass-upgrade? See if my latest change has what you had in mind.